src/app/layout/header/dialog-user/dialog-user.component.spec.ts

```diff
@@ -123,5 +124,5 @@ const selectors = (hostDebug: DebugElement) => {
 
 const todayDate = () => {
   const date = new Date(Date.now());
-  return `${date.getMonth()}/${date.getDate()}/${date.getFullYear()}`;
+  return `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()}`;
```

Comment:
Is this a bugfix? Or is this a refactor as it was being done elsewhere?

Reply:
No, this is a fix for bug inside a test. `getMonth()` gets value from 0 to 11, not 1 to 12. It was working until now - because getMonth returned value bigger than 0. So it created a valid date. Now it was returning 0/4/2022 - which is wrong.